frontend/plugin-manager/src/utils/request.ts

```
@@ -181,7 +182,12 @@ service.interceptors.response.use(
         case 404:
           message = formatHttpError(error) || i18n.global.t('messages.resourceNotFound')
           // 404 错误不显示通用错误消息，让调用方自己处理
-          ElMessage.closeAll()
+          const preserveMessagesOn404 = Boolean(
+            (error.config as ErrorDisplayRequestConfig | undefined)?.preserveMessagesOn404,
+          )
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)